app/user-list/user-list.component.js

```diff
@@ -1,15 +1,14 @@
 export default {
-    templateUrl: './app/user-list/user-list.template.html',
-    controller: ['$http',
-        function UserListController($http) {
-            var self = this;
-            $http({
-                method: 'GET',
-                url: 'http://localhost:3000/users'
-            }).then((response) =>
-                self.users = response.data
-            );
-            this.orderProp = '-regDate';
-        }
-    ]
+    template: require('./user-list.template.html'),
+    controller: function UserListController($http) {
+        'ngInject';
+        var self = this;
+        $http({
+            method: 'GET',
+            url: 'http://localhost:3000/users'
+        }).then((response) =>
+            self.users = response.data
+        );
+        this.orderProp = '-regDate';
```

Comment:
You saved this into self - use self then, and more common name for this in controllers is "vm"